### DISCUSSION TITLE
Zoom Meetings Time Change

### DISCUSSION THREAD

**Discussion by Joseph-Kassab** [DrAlzahraniProjects/csusb_fall2024_cse6550_team2#28] [Announcements]
The Professor is requiring us to meet Monday and Friday from 12pm-1pm for the Zoom meetings. We can have additional meetings throughout the week as well if requested.

**Comment by farheen-akhter-code** [comment]
Yes please I am not available from 12-1pm due to my on campus job . If possible please arrange an additional meeting.

**Reply by Joseph-Kassab** [reply]
I can arrange additional meetings, but I can't mark you on the progress spreadsheet for those. I made a poll for a new meeting time, but we can only reschedule if everyone agrees on the same time.

**Comment by JasonHernandez24** [comment]
Do we have to attend both meetings? I would only be able to attend the Monday meetings

**Reply by Joseph-Kassab** [reply]
It's up to the Professor, but I can write down your reason for missing the Friday meetings.
